test/model.js

```diff
@@ -428,27 +421,24 @@
   });
 
   QUnit.test("defaults", function(assert) {
-    assert.expect(4);
+    assert.expect(5);
     var Defaulted = Backbone.Model.extend({
       defaults: {
-        "one": 1,
-        "two": 2
+        one: 1,
+        two: 2
       }
     });
     var model = new Defaulted({two: undefined});
     assert.equal(model.get('one'), 1);
     assert.equal(model.get('two'), 2);
+    model = new Defaulted({two: 3});
+    assert.equal(model.get('one'), 1);
+    assert.equal(model.get('two'), 3);
     Defaulted = Backbone.Model.extend({
-      defaults: function() {
-        return {
-          "one": 3,
-          "two": 4
-        };
-      }
+      defaults: { hasOwnProperty: true, }
     });
-    model = new Defaulted({two: undefined});
```

Comment:
> Isn't that this test?

Sorry, missed that in the diff. 

> The only difference with mine is it won't respect properties that are already on the dest object

Then I'm confused. That changes _.default's behavior completely, no?